Code/Tools/ProjectManager/Source/ProjectManagerWindow.cpp

```diff
@@ -7,14 +7,23 @@
  */
 
 #include <ProjectManagerWindow.h>
+#include <PythonBindingsInterface.h>
 #include <ScreensCtrl.h>
 
 namespace O3DE::ProjectManager
 {
     ProjectManagerWindow::ProjectManagerWindow(QWidget* parent, const AZ::IO::PathView& projectPath, ProjectManagerScreen startScreen)
         : QMainWindow(parent)
     {
-        setWindowTitle(tr("O3DE Project Manager"));
+        if (auto engineInfoOutcome = PythonBindingsInterface::Get()->GetEngineInfo(); engineInfoOutcome)
+        {
+            auto engineInfo = engineInfoOutcome.GetValue<EngineInfo>();
+            setWindowTitle(QString("%1 %2 Project Manager").arg(engineInfo.m_name.toUpper(), engineInfo.m_version));
+        }
+        else
+        {
+            setWindowTitle("O3DE Project Manager");
```

Comment:
Was the removal of tr here intended?

Reply:
It was, but I think my original reason which was to avoid the engine and application name not matching what is in the docs, is not necessary and will add back the tr() so we can translate the app name in the future if we want.